wordpress-mobile/gutenberg-mobile
# Add capability to bridge to show/hide audio block

Adds capability to bridge to show/hide audio block

gutenberg PR: https://github.com/WordPress/gutenberg/pull/28952
WPAndroid PR: https://github.com/wordpress-mobile/WordPress-Android/pull/14037
WPiOS PR: https://github.com/wordpress-mobile/WordPress-iOS/pull/15865

To test:
1. Switch to a free/simple site
1. Open editor
1. Tap block inserter
1. Audio Block should not be in the list  
 
#
  
1. Switch to a paid/self-hosted site
1. Open editor
1. Tap block inserter
1. Audio Block should be in the list

PR submission checklist:
